internal/packetfabric/cloud_router_connection.go

```diff
@@ -74,6 +77,11 @@ type CloudRouterConnectionReadResponse struct {
 	Site                      string           `json:"site,omitempty"`
 }
 
+type BgpStateObj struct {
```

Comment:
Please use `gofmt` for code formatting. All popular editors support it, and it is part of standard Go tooling.